packages/build/src/steps/run_core_step.js

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable max-lines */
+import { getConfigOpts, loadConfig } from '../core/config.js'
+import { getConstants } from '../core/constants.js'
+import { normalizeFlags } from '../core/normalize_flags.js'
+import { getSeverity } from '../core/severity.js'
+import { handleBuildError } from '../error/handle.js'
+import { getErrorInfo } from '../error/info.js'
+import { startErrorMonitor } from '../error/monitor/start.js'
+import { getBufferLogs } from '../log/logger.js'
+import { logBuildStart } from '../log/messages/core.js'
+import { bundleEdgeFunctions } from '../plugins_core/edge_functions/index.js'
+import { reportStatuses } from '../status/report.js'
+
+import { runSteps } from './run_steps.js'
+
+/**
+ * Executes specified build steps for functions
```

Comment:
Small suggestion to make this comment more accurate (since we can run any core steps, not necessarily specific to functions) and more in line with the same comment in `src/core/main.js`.

```suggestion
 * Runs specific core steps for a build and returns whether it succeeded or not.
```